pkg/model/sdk_helper.go

```diff
@@ -93,8 +93,8 @@ func (h *SDKHelper) WithAPIVersion(apiVersion string) {
 }
 
 // API returns the aws-sdk-go API model for a supplied service alias
```

Comment:
nit: comment should be `... for a supplied service model name`. Same for below. Or `model service alias` depending on above discussion